src/include/duckdb/common/serializer/serialization_traits.hpp

```diff
@@ -10,6 +12,8 @@ namespace duckdb {
 class FormatSerializer;   // Forward declare
 class FormatDeserializer; // Forward declare
 
+typedef uint16_t field_id_t;
```

Comment:
Possibly bump this to `uint32_t`, just to have slightly more flexibility.

Reply:
Agree, done!